src/schemas/txpool.yaml

```diff
@@ -0,0 +1,50 @@
+TxPoolFilter:
+  title: txpool filter
+  type: object
+  properties:
+    from:
+      title: from address
+      $ref: '#/components/schemas/address'
+    to:
+      title: to address
+      $ref: '#/components/schemas/address'
+    gas:
+      title: gas limit
+      $ref: '#/components/schemas/FilterOperators'
+    gasPrice:
+      title: gas price
+      $ref: '#/components/schemas/FilterOperators'
+    value:
+      title: value
+      $ref: '#/components/schemas/FilterOperators'
+    nonce:
```

Comment:
It might not be used that much, but I can imagine if you have multiple txs pending (or stuck), it could be helpful to be able to filter. But this might be more with the idea that while we're at it we might as well include these